### ISSUE TITLE
Export tree exports to 3.4 tree

### ISSUE THREAD

**Issue by Potawto** [Openarl/PathOfBuilding#1311]
The passive tree exports to https://www.pathofexile.com/passive-skill-tree/3.4.0/XXXXXXX instead of https://www.pathofexile.com/passive-skill-tree/3.5.0/XXXXXXX

**Comment by Exportforce** [comment]
yeah this happens every single patch.  openarl should take out the version number from the export at all. 
If you enter no number its just the most recent tree which everyone wants. also it should lead to https://www.pathofexile.com/fullscreen-passive-skill-tree/ instead
